pkg/kubectl/cmd/cmd_test.go

```diff
@@ -354,8 +360,8 @@ func Example_printPodWithWideFormat() {
 		fmt.Printf("Unexpected error: %v", err)
 	}
 	// Output:
-	// NAME      READY     STATUS     RESTARTS   AGE       IP         NODE
-	// test1     1/2       podPhase   6          10y       10.1.1.3   kubernetes-node-abcd
+	// NAME      READY     STATUS     RESTARTS   AGE       IP         NODE                   REFERENCE
+	// test1     1/2       podPhase   6          10y       10.1.1.3   kubernetes-node-abcd   ReplicationController/my-rc
```

Comment:
can i add the field in -o wide? this fiele can help me clean uo my po. sometimes write bash because easy.